src/Microsoft.OpenApi/Services/OpenApiWalker.cs

```diff
@@ -1036,6 +1037,7 @@ internal void Walk(IOpenApiElement element)
                 case OpenApiOAuthFlow e: Walk(e); break;
                 case OpenApiOperation e: Walk(e); break;
                 case OpenApiParameter e: Walk(e); break;
+                case OpenApiPaths e: Walk(e); break;
```

Comment:
We already walk OpenAPIPaths here https://github.com/microsoft/OpenAPI.NET/blob/e6535f7380e424668163f812da80d8e8c270b649/src/Microsoft.OpenApi/Services/OpenApiWalker.cs#L48   I think this line may not be necessary.  The Visit method in the OpenAPIValidator is main thing that is needed.

Reply:
I was wrong.  This is used for validating fragments.